src/mame/video/rad_hsvpal.h

```diff
@@ -0,0 +1,67 @@
+// license:BSD-3-Clause
+// copyright-holders:David Haywood
+
+// this is based on emupal.h, style etc. (including use of statics) copied where possible.  If considered generic enough could probably be merged?
+
+#ifndef MAME_VIDEO_RAD_HSVPAL_H
+#define MAME_VIDEO_RAD_HSVPAL_H
+
+class radica_hsvpal_device : public palette_device
+{
+public:
+	radica_hsvpal_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
+
+	template<int _HueMask, int _SaturationMask, int _ValueMask, int _HueShift, int _SaturationShift, int _ValueShift, int _V_max>
```

Comment:
Identifiers beginning with an underscore followed by a capital letter are reserved.  Please remove the leading underscores.